spec/models/reservation_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Reservation, type: :model do 
+	before(:each) do
+      @reservation = create(:reservation)
```

Comment:
The text editor will have options to interconvert tabs and spaces (as well as their sizes).